NPS_generation/commands/create_training_sets.py

```diff
@@ -66,17 +65,16 @@ def get_similar_smiles(input_smiles, min_tc, n_molecules=100, max_tries=200, see
 
     # shuffle SMILES and fingerprints
     inputs = list(zip(input_smiles, input_fps))
-    random.seed(seed)
-    random.shuffle(inputs)
+    np.random.shuffle(inputs)
```

Comment:
I'm using numpy instead of the `random` module wherever possible, to reduce the number of moving parts (`np.seed` can be set and everything downstream from there becomes deterministic).